exercises/ex3.rb

```diff
@@ -0,0 +1,48 @@
+class Geometry
+  attr_accessor :x, :y
+
+  def initialize(x, y)
+    @x = x
+    @y = y
+  end
+
+  def distance(point)
+    Math.sqrt((@x - point.x) ** 2 + (@y - point.y) ** 2)
+  end
+
+  def middle(point)
+    @mid_x = ((@x + point.x).to_f) / 2
+    @mid_y = ((@y + point.y).to_f) / 2
```

Comment:
hạn chế dùng `()` khi không cần thiết
```suggestion
    @mid_x = (@x + point.x).to_f / 2
    @mid_y = (@y + point.y).to_f / 2
```